generator/src/lib.rs

```diff
@@ -1109,9 +1109,13 @@ fn generate_function_pointers<'a>(
     let loader = commands.is_empty().not().then(|| {
         quote! {
             impl #ident {
-                pub fn load<F>(mut _f: F) -> Self
+                pub fn load<F>(mut f: F) -> Self
                     where F: FnMut(&CStr) -> *const c_void
                 {
```

Comment:
How bad would it be to shorten these lines a bit:

```suggestion
                pub fn load<F: FnMut(&CStr) -> *const c_void>(mut f: F) -> Self {
```

Or to look even more similar to the `dyn` below, since `F` is never referenced:

```suggestion
                pub fn load(mut f: impl FnMut(&CStr) -> *const c_void) -> Self {
```

Reply:
Went with the former, because the latter is a breaking change. Specifying `F` explicitly probably isn't all that useful, but no harm in preserving the capability.